modules/Crowd Assessment/moduleFunctions.php

```diff
@@ -311,7 +311,7 @@ function getThread($guid, $connection2, $gibbonPlannerEntryHomeworkID, $parent,
             $output .= "<a name='".$rowDiscuss['gibbonCrowdAssessDiscussID']."'></a>";
             $output .= "<table class='noIntBorder chatBox $classExtra' cellspacing='0' style='width: ".(755 - ($level * 15)).'px; margin-left: '.($level * 15)."px'>";
             $output .= '<tr>';
-            $output .= '<td><i>'.formatName($rowDiscuss['title'], $rowDiscuss['preferredName'], $rowDiscuss['surname'], $rowDiscuss['category']).' said</i>:</td>';
+            $output .= '<td><i>'.formatName($rowDiscuss['title'], $rowDiscuss['preferredName'], $rowDiscuss['surname'], $rowDiscuss['category']).' '.__('said').'</i>:</td>';
```

Comment:
Please use the string replacement syntax if possible. It would make the strings more readable for the translators and provide more flexibilities:

```php
__('{name} said', [
  'name' => formatName($rowDiscuss['title'], $rowDiscuss['preferredName'], $rowDiscuss['surname'], $rowDiscuss['category'])
])
```

Reply:
Thanks for your feedback. I'll prepare one PR with these improvements.

Reply:
Hi Leo. Are you Ok to push the suggested change for Crowd Assessment/moduleFunctions.php  line 314 to this PR? Then I think it's ready to merge. Thanks!

Reply:
Hi Sandra, I removed this change and added it in PR #807. I wait for the merge to add the strings in the language package.

Reply:
Thanks!